db/migrate/20220720214646_add_delete_cascade_from_orgs_down.rb

```diff
@@ -0,0 +1,11 @@
+class AddDeleteCascadeFromOrgsDown < ActiveRecord::Migration[7.0]
+  def up
+    add_foreign_key :schemes, :organisations, column: "owning_organisation_id", on_delete: :cascade
```

Comment:
You're adding the same key twice here, and removing it twice below which isn't right.

Best way is to run the migration up and down locally and check it gives the expected schema in both cases.